### PR TITLE
[Analytics, MT & MWAN] Added new GraphQL section for tunnel bandwidth analytics

### PR DESCRIPTION
**Pull request by dcpena**
Added new GraphQL section for Magic Transit tunnel bandwidth analytics to address [PCX-3671](https://jira.cfops.it/browse/PCX-3671).

### CODE REVIEW

**Review by tlozoot — APPROVED** [review]

File: content/analytics/graphql-api/tutorials/querying-magic-transit-tunnel-bandwidth-analytics/index.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
+---
+pcx-content-type: tutorial
+title: Querying Magic Transit Tunnel Bandwidth Analytics with GraphQL
+---
+
+# Querying Magic Transit Tunnel Bandwidth Analytics with GraphQL
```
Comment (marciocloudflare) [inline content/analytics/graphql-api/tutorials/querying-magic-transit-tunnel-bandwidth-analytics/index.md]: Do wee need everything in capital letters? Could it be `Querying Magic Transit tunnel bandwidth analytics with GraphQL`?

File: content/analytics/graphql-api/tutorials/querying-magic-transit-tunnel-bandwidth-analytics/index.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+In this example, you are going to use the GraphQL Analytics API to query Magic Transit Ingress Tunnel Traffic over a specified time period.
+
+The following API call will request Magic Transit Ingress Tunnel Traffic over a one hour period and output the requested fields. Be sure to replace `CLOUDFLARE_ACCOUNT_ID`, `CLOUDFLARE_EMAIL`, and `CLOUDFLARE_API_KEY` with your email and API credentials, and adjust the `datetime_geg` and `datetime_leq `values as needed.
```
Comment (marciocloudflare) [inline content/analytics/graphql-api/tutorials/querying-magic-transit-tunnel-bandwidth-analytics/index.md]:
```suggestion
The following API call will request Magic Transit Ingress Tunnel Traffic over a one-hour period and output the requested fields. Be sure to replace `CLOUDFLARE_ACCOUNT_ID`, `CLOUDFLARE_EMAIL`, and `CLOUDFLARE_API_KEY` with your email and API credentials, and adjust the `datetime_geg` and `datetime_leq `values as needed. 
```

File: content/analytics/graphql-api/tutorials/querying-magic-transit-tunnel-bandwidth-analytics/index.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+title: Querying Magic Transit Tunnel Bandwidth Analytics with GraphQL
+---
+
+# Querying Magic Transit Tunnel Bandwidth Analytics with GraphQL
+
+In this example, you are going to use the GraphQL Analytics API to query Magic Transit Ingress Tunnel Traffic over a specified time period.
+
+The following API call will request Magic Transit Ingress Tunnel Traffic over a one hour period and output the requested fields. Be sure to replace `CLOUDFLARE_ACCOUNT_ID`, `CLOUDFLARE_EMAIL`, and `CLOUDFLARE_API_KEY` with your email and API credentials, and adjust the `datetime_geg` and `datetime_leq `values as needed. 
+
+The following example queries for ingress traffic. To query for egress, then change the value in the direction filter.
```
Comment (marciocloudflare) [inline content/analytics/graphql-api/tutorials/querying-magic-transit-tunnel-bandwidth-analytics/index.md]:
```suggestion
The following example queries for ingress traffic. To query for egress, change the value in the direction filter.
```

File: content/analytics/graphql-api/tutorials/querying-magic-transit-tunnel-bandwidth-analytics/index.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+curl \
+  -X POST \
+  -H "Content-Type: application/json" \
+  -H "X-Auth-Email: CLOUDFLARE_EMAIL" \
+  -H "X-Auth-key: CLOUDFLARE_API_KEY" \
+  --data "$(echo $PAYLOAD)" \
+  https://api.cloudflare.com/client/v4/graphql/
+  ```
+
+The returned values represent the total bandwidth in bits/s during the five minute interval for a particular tunnel. To use aggregations other than five minutes, make sure that you use the same window for both your metric and date time. For example, to see hourly groups, use `bitRateHour` and `datetimeHour`.
```
Comment (marciocloudflare) [inline content/analytics/graphql-api/tutorials/querying-magic-transit-tunnel-bandwidth-analytics/index.md]:
```suggestion
The returned values represent the total bandwidth in bits/second during the five minute interval for a particular tunnel. To use aggregations other than five minutes, make sure that you use the same window for both your metric and date time. For example, to see hourly groups, use `bitRateHour` and `datetimeHour`.
```

File: content/magic-wan/how-to/view-analytics.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
+---
+title: View Network Analytics
+pcx-content-type: how-to
+weight: 4
+---
+
+# View Network Analytics
```
Comment (marciocloudflare) [inline content/magic-wan/how-to/view-analytics.md:7]: Same question: could it be `View network analytics`? Is it the name of a product? Asking as below it's also referred as `Network Analytics`

**Review by marciocloudflare — APPROVED** [review]

File: content/magic-wan/how-to/view-analytics.md (hunk 5 above)
Comment (dcpena) [reply content/magic-wan/how-to/view-analytics.md:7]: It's not a product, but it is a specific portion of the dash.

File: content/analytics/graphql-api/tutorials/querying-magic-transit-tunnel-bandwidth-analytics/index.md (hunk 1 above)
Comment (dcpena) [reply content/analytics/graphql-api/tutorials/querying-magic-transit-tunnel-bandwidth-analytics/index.md]: You are absolutely right! Thanks for the catch

**Pull request merged**